backend/oasst_backend/models/text_labels.py

```diff
@@ -15,6 +15,9 @@ class TextLabels(SQLModel, table=True):
             pg.UUID(as_uuid=True), primary_key=True, default=uuid4, server_default=sa.text("gen_random_uuid()")
         ),
     )
+    user_id: UUID = Field(
+        sa_column=sa.Column(pg.UUID(as_uuid=True), sa.ForeignKey("user.id"), nullable=False, primary_key=True)
```

Comment:
could it be `primary_key=True` here is a mistake?

Reply:
Yep good spot, removed